test/README.md

```diff
@@ -0,0 +1,53 @@
+# Tests
+
+## Structures
+
+There are 2 suites of tests here: [`e2e`](./e2e) and [`osd`](./osd). 
+
+* [`e2e`](./e2e)
+  
+  The tests in this directory are executed using the operator-sdk's test command against an OCP environment, and will be exectued everytime when a PR is opened.
+ 
+* [`osd`](./osd)
+
+  The tests in this directory are executed from inside of a container, as part of the [OSD Addon testing flow](https://github.com/openshift/osde2e/blob/master/docs/Addons.md), against an OSD environment. 
+  Currently these tests are executed periodically (every 4 hours).
+  
+Even though the test suites are executed against different environments, it should be possible to reuse some of the code. That's what the [`common`](./common) directory is for.
```

Comment:
Maybe this answers my question, but i would expect the majority of tests to be common